Dockerfile

```diff
@@ -10,6 +11,26 @@ RUN ./node_modules/.bin/gulp build
 RUN npm run lint:js -- --max-warnings=0
 RUN npm run lint:css
 
+FROM debian as react-builder
+
+# Install Volta, to use the same toolchain (Node+npm) as developers locally:
+RUN apt-get update && apt-get install -y curl ca-certificates --no-install-recommends
+ENV VOLTA_HOME $HOME/.volta
+ENV PATH $VOLTA_HOME/bin:$PATH
+RUN curl https://get.volta.sh | bash
```

Comment:
This ensures our CI has the same toolchain (i.e. Node and npm version) as we do locally, but if we don't trust the Volta installer to run in CI, we can also just use the Node Docker image and manually make sure it's the same version.

Reply:
Yeah, though of course using a Docker image and installing npm dependencies is _already_ doing remote code execution with access to our secrets :/ 

I'll try to remove this in a separate branch (so I can test), and post back here when I've done that.